tests/integration-service/integration.go

```diff
@@ -118,6 +135,11 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 				}
 				return nil
 			}, timeout, interval).Should(Succeed(), fmt.Sprintf("timed out when trying to check if the Snapshot for component %s/%s exists", testNamespace, componentName))
+			pipelineRun, err := f.AsKubeAdmin.IntegrationController.GetBuildPipelineRun(componentName, applicationName, testNamespace, false, "")
+			if err != nil {
+				GinkgoWriter.Printf("PipelineRun for Component %s/%s can't be gotten successfully\n", testNamespace, componentName)
+			}
+			Expect(pipelineRun.Labels["appstudio.openshift.io/snapshot"]).To(Equal(snapshot.GetName()))
```

Comment:
Also, I think we're adding the snapshot's name as an "annotation" to the build PLR rather than as a "label".
If I'm not wrong, this is the reason why the latest CI run failed on both of these tests on the following error ([look here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/628/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1681283386255085568#1:build-log.txt%3A1215)):
```
[FAILED] Expected
      <string>: 
  to equal
      <string>: mvp-test-app-8cfrh
```

Reply:
It's a label which is decided by the source code, not the test case.

Reply:
The second one is for IntegrationPipelineRun. It's an Annotation in Build PipelineRun.